pkg/persistence/config/loader/config_loader_test.go

```diff
@@ -1323,6 +1324,117 @@ configs:
         wrong-field: var`,
 			wantErrorsContain: []string{"missing property"},
 		},
+		{
+			name: "Document dashboard config with FF on",
```

Comment:
The name "... with FF on" suggests to me that it will also be a test "with FF off". I would argue (and no test with FF off exists) to remove that from the name.

Reply:
A couple of tests down there is one with the feature flag off that expects an error to be generated